projects/valtimo/components/src/lib/components/dropzone/dropzone.component.ts

```diff
@@ -142,7 +142,10 @@ export class DropzoneComponent implements OnInit, AfterViewInit, OnDestroy {
     this.dropzone.on('error', (file, message) => {
       this.clearFile();
       this.error$.next(message.toString());
-      this.fileSelected.emit();
+
+      setTimeout(() => {
+        this.clearError();
+      }, 7500);
     });
```

Comment:
Dus na 7,5 seconden wordt altijd een mogelijke error message verwijderd? Waarom specifiek die 7,5 seconden?

Reply:
Gevoelskwestie; 5 voelde te kort 10 te lang. Kan het instelbaar maken maar vond ik een beetje overkill

Reply:
Voelt niet als een top oplossing.
Is de foutmelding niet te tonen en dan de rest van de functionaliteit heel te houden.

Dus dat je de foutmelding ziet maar dan een andere file kan uploaden. Op het moment dat er een nieuwe file wordt aangeboden dat je dan de foutmelding van de vorige file cleared?

Reply:
Alright heb die timeout weggehaald, werkt nu zoals jij hierboven omschrijft